node-deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM circleci/node:12.16
+FROM cimg/node:16.1.0
```

Comment:
jfyi: i'm planning to switch to a more secure `distroless/nodejs` docker-image provided by google. but i have just checked it, they dont yet have v16 avail.

Reply:
Understood. Let's keep that on our radar.